code/modules/sealant_gun/sealant_rack.dm

```diff
@@ -0,0 +1,62 @@
+/obj/structure/sealant_rack
+	name = "sealant tank rack"
+	icon = 'icons/obj/structures/sealant_props.dmi'
+	icon_state = "rack"
+	var/obj/item/gun/launcher/sealant/loaded_gun
+	var/list/tanks
```

Comment:
Clear this stuff on Destroy.

Reply:
Not sure how I missed it for these, usually I'm good about it!